pkg/framework/util.go

```diff
@@ -31,9 +31,12 @@ import (
 )
 
 const (
-	defaultUIDPrefix       string = "e2e-cri-uid"
-	defaultNamespacePrefix string = "e2e-cri-namespace"
-	defaultAttempt         uint32 = 2
+	defaultUIDPrefix            string = "e2e-cri-uid"
+	defaultNamespacePrefix      string = "e2e-cri-namespace"
+	defaultAttempt              uint32 = 2
+	defaultContainerImage       string = "gcr.io/google_containers/busybox:1.24"
```

Comment:
> I think runtime should pull the image if it does not exist. 

No, this is not consistent with CRI. There is explicit image pull API, which ensures the image is pulled before creating containers. We should not rely on docker's behavior.